tests/pipelines/stable_diffusion_xl/test_stable_diffusion_xl.py

```diff
@@ -1088,6 +1088,10 @@ def test_single_file_component_configs(self):
         for param_name, param_value in single_file_pipe.unet.config.items():
             if param_name in PARAMS_TO_IGNORE:
                 continue
+            if param_value == "upcast_attention" and pipe.unet.config[param_name] is None:
+                print(f"Pipe attrbiute {pipe.unet.config[param_name]}")
```

Comment:
Why print?

Reply:
Sorry that was a debugging residue. Should now be all fixed.